data/data/openstack/main.tf

```diff
@@ -52,10 +52,12 @@ module "masters" {
   master_sg_ids = concat(
     var.openstack_master_extra_sg_ids,
     [module.topology.master_sg_id],
+    var.openstack_additional_security_groups_ids,
```

Comment:
What's the difference between `openstack_master_extra_sg_ids` and `openstack_additional_security_groups_ids` variables?

Reply:
>_< oh my... we already support this.